src/docfx.website.themes/default/token.json

```diff
@@ -34,5 +34,10 @@
   "eventType": "Event Type",
   "exceptions": "Exceptions",
   "condition": "Condition",
-  "extensionMethods": "Extension Methods"
+  "extensionMethods": "Extension Methods",
+  "note": "<h5>Note</h5>",
```

Comment:
MardownRenderer doesn't know the proper heading level in page. We allow token.json in template to custommize it.